### ISSUE TITLE
Two Problems with dcl:092

### ISSUE THREAD

**Issue by smallfind** [pulibrary/cicognara-catalogo#132]
Hi folks,

Looks like images for this record: https://figgy.princeton.edu/catalog/72244e8d-00da-4d21-ba5a-9b76fa238fce ~ should actually be attached to PUL Bib/ Source Metadata Identifier 8544151 (https://catalog.princeton.edu/catalog/8544151) and not Source Metadata Identifier 10084786.

However, this item (https://figgy.princeton.edu/catalog/ae79ee25-240f-4388-83a7-1f054bbec647) has two sets of local dentifiers listed: dcl:092 in the imported attributes and then also dcl:096, dcl:337, and dcl:p02. 

Note that dcl:092 appears attached to both titles in the spreadsheet the student team and I are working on.

Any thoughts?

Cheers,

-- d

**Comment by escowles** [comment]
@smallfind I agree  https://figgy.princeton.edu/catalog/72244e8d-00da-4d21-ba5a-9b76fa238fce looks like 8544151, so you should be able to edit that in Figgy and update the remote metadata to pull in the correct metadata (and get the object published on the right page in the Catalog).

For https://figgy.princeton.edu/catalog/ae79ee25-240f-4388-83a7-1f054bbec647, the three DCL numbers in the "Local Identifier" field are imported from the MARC record, and the "dcl:092" one is manually entered. Maybe it's a typo and should be dcl:096 or dcl:p02?

**Comment by smallfind** [comment]
OK. This is my first attempt at this, sooooo ... how does this look now? (https://figgy.princeton.edu/catalog/72244e8d-00da-4d21-ba5a-9b76fa238fce)

(I'm looking into the second now.)

**Comment by smallfind** [comment]
OK, yeah -- looking for the bib # now pulls up three separate results and it's possible to identify where the typo happened by crosschecking the imported and the manually entered identifiers. You were right. Thanks for thinking of this!

**Comment by escowles** [comment]
@smallfind Looks good — the viewer now shows up for that first record: https://catalog.princeton.edu/catalog/8544151 (and glad the simple low-tech typo idea turned out to be right!)

**Comment by lightgivener** [comment]
Hey guysy, the dcl:092 appears twice in the ShotList. I had assigned the new dcl:zzh to Le Bellezze di Verona Cico No 4388.

Since I have 87 new DCL numbers assigned in my extra spreadsheet for that purpose I should probably pass that along sooner than later if you are working so much on it at the moment. I still have some hours left and am working with Ani through problems so there might some additional ones coming later as well.


OLD SHOTLIST: Title | Creator | # of Fiche | # of Images | Box # | Fiche Nr. | Catalogo Nr | PUL Bib | DCL Nr | Files Dir.
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
Christianrum   antiquitatum specimina.., 1758 | 2 | 59 | 34 | 3215 (3) | 3215 | 10084786 | 092 | 092
Le bellezze di Verona | d'Adriano Valerini ... ; nel quale con breuità si tratta di   tutte le cose notabili della città ... | 2 | 65 | 47 | 4388 | 4388 | 8544151 | 092 | 092


The Christianum Antiquitatum title has three copies in the Vatican all on fiche so that explains the three dcl's. There is however the typo with the 096 which i don't have anwhere. The correct dcl's for those three Christinum Antiquitatum fiche are dcl:092, dcl:337 and dcl:p02. I'll let Bary know about this, too.

MASTER FILE:

Cico-Nr. | Double Number | Author | Title | Variant Title | Place | Year | OCLC NR ANI | Holding   Institution | To be Digitized | Online all   institutions | Vatican call no. | Further Holding   Institution | Heidelberg /   Shelfmark | Princeton /        Location | OCLC (Getty / MMS   ID) | NGA Operator   Searched | Harvard /     LOC | Cico-Nr. /with   combined double Nr. | Missing Numbers   added | Cico-Nr. /   Original  (as on fiche header) | Total Number of   Works | Shot List Fiche |   | DCL Nr
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
3215-3 | 3245   / 3247 | Jacutius,   Matthaeus | Matthaei   Jacvtii ... Christianarvm Antiqvitatvm Specimina Qvae In Vetere Bonvsae Et   Mennae Titvlo E Svbvrbana Agathae Basilica Ann. MDCCLVII. Vaticanvm Ad Mvsevm   Transvecto Exercitationibvs Philologicosacris Nec Non Praecipvis Basilicae   Eivs Inclytae Monvmentis Atqve Ineditis Inscriptionibvs Variis Collvstrantvr |   | Romae   : Monaldini | 1758 |   | NGA |   | 0 | Cicognara.IV.3215(int.3) |   |   |   |   | VK |   | 3215(3),   3245, 3247 |   | 3215(3) | 0 | 3215 (3) | ## | 092
3245 | 3215-3   / 3247 | Jacutius,   Matthaeus | [Commentarium in titulum Bonusae et Mennae] Matthaei Jacutii benedictini congregationis montisvirginis   Christanarum antiquitatum specimina, quae in vetere Bonusae et Mennae titulo   e subrana Agathae basilica |   | Roma | 1758 |   | NGA |   | 0 | Cicognara.VI.M.3(int.3) | BSB   / KHI / ZI |   |   |   | rare |   | 3245,   3247, 3215(3) | added | [fiched as] 3247  = VI M 3(3) | 0 |   |   | 337
3247 | 3215-3   / 3245 | Jacutius,   Matthaeus | Matthæi   Jacvtii ... Christianarvm Antiqvitatvm Specimina Qvæ In Vetere Bonvsæ Et   Mennæ Titvlo E Svbvrbana Agathæ Basilica Ann. MDCCLVII. Vaticanvm Ad Mvsevm   Transvecto Exercitationibvs Philologicosacris Nec Non Præcipvis Basilicæ Eivs   Inclytæ Monvmentis Atqve Ineditis Inscriptionibvs Variis Collvstrantvr |   | Romæ   : Monaldini | 1758 |   | NGA |   | 1 | Cicognara.IV.3247(int.1) |   |   |   |   | VK |   | 3247,   3245 3215(3) |   | 3247(1) | 1 |   |   | p02

**Comment by smallfind** [comment]
Would it be easier to change what you have as dcl:092 (3215-3/ Matthaei Jacvtii ... Christianarvm Antiqvitatvm) to dcl:096? And change dcl:zzh / Le Bellezze di Verona Cico / No 4388 back to dcl:092?

My concern is that the dcl:096 now occurs across multiple records? (Or is there a simple fix in the system that I'm unaware of?)

**Comment by lightgivener** [comment]
dcl:096 shouldn't appear anywhere?

It is not anywhere in the ShotList nor in the master file. I thought it was just the typo in the MARC record?

Is dcl:096 assigned to any other fiche scans at the moment?

Huh hum. I'll have to check in with Bary about this. Maybe it is possible as you suggested but please let me know if dcl:096 appears on anything else than the Christianarum title.

**Comment by lightgivener** [comment]
Alright @smallfind I did just that and changed the master file around the now existing dcl identifiers.

So the three copies of Christianarum antiquitatum are: 
dcl:096 = fiche header: 3215(3)
dcl:337 = fiche header: [fiched as] 3247  = VI M 3(3)
dcl:p02 = fiche header: 3247(1)

and the Bellezze di Verona is now dcl:092.

Hurts my sould but should now hopefully work out.

This means I also had to adjust the NEW DCL Numbers file and have reuploaded into the Google Drive.

Cheers!

**Comment by lightgivener** [comment]
master file excerpt for completeness: 

Cico-Nr. | Double   Number | Author | Title | Place | Year | Vatican   call no. | original Catalogo mistake | Double   Number category |   | Cico-Nr. /with combined double Nr. | Cico-Nr. / Original  (as on fiche   header) | Total   Number of Works | Shot List   Fiche |   | DCL Nr
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
3215-3 | 3245 / 3247 | Jacutius,   Matthaeus | Matthaei Jacvtii   ... Christianarvm Antiqvitatvm Specimina | Romae :   Monaldini | 1758 | Cicognara.IV.3215(int.3) |   | identical   edition? | plates in   different order | 3215(3),   3245, 3247 | 3215(3) | 0 | 3215 (3) | ## | 096
3245 | 3215-3 /   3247 | Jacutius,   Matthaeus | [Commentarium in titulum Bonusae et Mennae] Matthaei Jacutii   benedictini congregationis montisvirginis Christanarum antiquitatum | Roma | 1758 | Cicognara.VI.M.3(int.3) | fiche header number was wrongly assigned | identical   edition? |   | 3245, 3247,   3215(3) | [fiched as] 3247  = VI M   3(3) | 0 |   |   | 337
3247 | 3215-3 /   3245 | Jacutius,   Matthaeus | Matthæi Jacvtii   ... Christianarvm Antiqvitatvm Monvmentis Atqve Ineditis Inscriptionibvs   Variis Collvstrantvr | Romæ :   Monaldini | 1758 | Cicognara.IV.3247(int.1) |   | identical   edition? | plates in   different order | 3247, 3245   3215(3) | 3247(1) | 1 |   |   | p02
